examples/tabular/example_quantile_regression.py

```diff
@@ -1,24 +1,38 @@
 """ Example script for quantile regression with tabular data, demonstrating simple use-case """
-
+import numpy as np
 from autogluon.tabular import TabularDataset, TabularPredictor
 
 # Training time:
 train_data = TabularDataset('https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv')  # can be local CSV file as well, returns Pandas DataFrame
-train_data = train_data.head(500)  # subsample for faster demo
+train_data = train_data.head(1000)  # subsample for faster demo
 print(train_data.head())
 
 label = 'age'  # specifies which column do we want to predict
 save_path = 'ag_models/'  # where to save trained models
-quantiles_topredict = [0.1, 0.5, 0.9]  # which quantiles of numeric label-variable we want to predict
+quantile_levels = [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9]  # which quantiles of numeric label-variable we want to predict
+num_quantiles = len(quantile_levels)
 
-predictor = TabularPredictor(label=label, path=save_path, problem_type='quantile', quantile_levels=quantiles_topredict)
-predictor.fit(train_data, time_limit=30)  # time_limit is optional, you should increase it for real applications
+predictor = TabularPredictor(label=label, path=save_path, problem_type='quantile', quantile_levels=quantile_levels)
+predictor.fit(train_data, calibrate=True, num_bag_folds=5) # calibration (conformalization)
 
 # Inference time:
 test_data = TabularDataset('https://autogluon.s3.amazonaws.com/datasets/Inc/test.csv')  # another Pandas DataFrame
```

Comment:
predict() took a long time for me, so consider subsampling test data too for faster demo